### DISCUSSION TITLE
RustDesk

### DISCUSSION THREAD

**Discussion by thejaceman** [runtipi/runtipi-appstore#2096] [Requests]
RustDesk is a full-featured open source remote control, TeamViewer alternative, application with minimal configuration.

**Links**
- Official repository: [https://github.com/rustdesk/rustdesk]
- Server Repository: [https://github.com/rustdesk/rustdesk-server]
- Official docker image: [https://hub.docker.com/r/rustdesk/rustdesk-server/]
- Official Website: [https://rustdesk.com/]

**Checklist**
- [x] I have put the app name in the title of the issue
- [x] I have linked the repository of the app
- [x] I have linked the official docker image

**Comment by thejaceman** [comment]
I now see there is already a request for RustDesk. I missed it beforehand. My apologies.

**Reply by JigSawFr** [reply]
Dup of https://github.com/runtipi/runtipi-appstore/discussions/2132
